packages/f-content-cards/src/components/ContentCards.vue

```diff
@@ -539,6 +542,17 @@ export default {
             return groupIndex !== null ?
                 this.testId && `ContentCard-${this.testId}-${index}-${groupIndex}` :
                 this.testId && `ContentCard-${this.testId}-${index}`;
+        },
+
+        /**
+         * Handles logging from f-metadata (callback)
+         * @returns {function(*, *=, *=): void}
+         */
+        handleLogging (logger) {
+            // eslint-disable-next-line func-names
+            return function (type, logMessage, payload) {
+                if (logger && logger[type]) logger[type](logMessage, null, payload);
+            };
```

Comment:
So you're meaning something like this, @DamianMullins? I don't see a problem with this, unless theres something else that precluded this, @matthewhardern ?
```suggestion
            return (type, logMessage, payload) => {
                if (logger && logger[type]) logger[type](logMessage, null, payload);
            };
```

Reply:
Exactly @benwhite-deltas

Reply:
changed as agreed, thought @DamianMullins meant remove the function lol, misread